rest/authorization-server/src/main/java/org/eclipse/sw360/rest/authserver/Sw360AuthorizationServer.java

```diff
@@ -11,60 +11,64 @@
 import org.eclipse.sw360.rest.common.Sw360CORSFilter;
-
-import org.apache.log4j.Logger;
-import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.builder.SpringApplicationBuilder;
 import org.springframework.boot.web.support.SpringBootServletInitializer;
 import org.springframework.context.annotation.Import;
 
-import javax.crypto.SealedObject;
-
-import java.io.IOException;
-import java.util.Properties;
-
-import static org.eclipse.sw360.rest.authserver.security.Sw360SecurityEncryptor.encrypt;
-
 @SpringBootApplication
 @Import(Sw360CORSFilter.class)
 public class Sw360AuthorizationServer extends SpringBootServletInitializer {
 
     private static final Logger log = Logger.getLogger(Sw360AuthorizationServer.class);
 
-    private static final String PROPERTIES_FILE_PATH = "/sw360.properties";
+    private static final String SW360_PROPERTIES_FILE_PATH = "/sw360.properties";
     private static final String DEFAULT_WRITE_ACCESS_USERGROUP = UserGroup.SW360_ADMIN.name();
     private static final String DEFAULT_ADMIN_ACCESS_USERGROUP = UserGroup.SW360_ADMIN.name();
 
+    private static final String SPRING_CONFIG_LOCATIION_KEY = "spring.config.location";
+    private static final String APPLICATION_PROPERTIES_DIRECTORY = "file:/etc/sw360/authorization/";
+
     public static final String CONFIG_ACCESS_TOKEN_VALIDITY_SECONDS;
     public static final UserGroup CONFIG_WRITE_ACCESS_USERGROUP;
     public static final UserGroup CONFIG_ADMIN_ACCESS_USERGROUP;
 
     static {
-        Properties props = CommonUtils.loadProperties(Sw360AuthorizationServer.class, PROPERTIES_FILE_PATH);
+        Properties props = CommonUtils.loadProperties(Sw360AuthorizationServer.class, SW360_PROPERTIES_FILE_PATH);
         CONFIG_WRITE_ACCESS_USERGROUP = UserGroup.valueOf(props.getProperty("rest.write.access.usergroup", DEFAULT_WRITE_ACCESS_USERGROUP));
         CONFIG_ADMIN_ACCESS_USERGROUP = UserGroup.valueOf(props.getProperty("rest.admin.access.usergroup", DEFAULT_ADMIN_ACCESS_USERGROUP));
         CONFIG_ACCESS_TOKEN_VALIDITY_SECONDS = props.getProperty("rest.access.token.validity.seconds", null);
     }
 
-    private static SealedObject getConfigClientSecret(Properties props) {
-        try {
-            return encrypt(props.getProperty("rest.security.client.secret", null));
-        } catch (IOException e) {
-            log.error("Error occured while encrypting client password", e);
-            return null;
-        }
-    }
-
     @Override
     protected SpringApplicationBuilder configure(SpringApplicationBuilder builder) {
-        return builder.sources(Sw360AuthorizationServer.class);
+        Properties properties = new Properties();
+
+        if(System.getProperty(SPRING_CONFIG_LOCATIION_KEY) == null) {
+            properties.setProperty(SPRING_CONFIG_LOCATIION_KEY, APPLICATION_PROPERTIES_DIRECTORY);
```

Comment:
Maybe move this code to a common ancestor class for `Sw360AuthorizationServer.java` and `Sw360ResourceServer.java`?